.github/workflows/pr.yml

```diff
@@ -78,13 +78,13 @@ jobs:
         with:
           args: '-v --release --out Lcov'
       - name: Push code coverage results to coveralls.io
-        uses: coverallsapp/github-action@master
+        uses: coverallsapp/github-action@main
```

Comment:
This will still need to be `master` I think, though you could consider just removing this job as I don't think we're doing anything with code coverage.